integration-test/helpers/harness.js

```diff
@@ -48,6 +55,12 @@ function withStringReplacements (page, replacements, platform = 'macos') {
         output = output.replace(`// INJECT ${keyName} HERE`, `${keyName} = ${replacement};`)
     }
 
+    if (constants) {
+        for (let [keyName, value] of Object.entries(constants)) {
+            output = output.replace(new RegExp(`${keyName}: \\d+`), `${keyName}: ${value}`)
+        }
+    }
```

Comment:
We're manipulating the source code with string substitution. A bit brutal but does the job. What's your thought on moving this to the remote config, so we can tweak it remotely if needed?

Reply:
yeah, I think better remote config integration is a must (as I've mentioned before given the advancements in C-S-S with this stuff)

But for this PR we shouldn't block on it. we do lots of replacements elsewhere and this is just for testing purposes so I see no issue.

Let's have a separate conversation about simplifying our 'Settings' class, now that we've been using it loads and we know what we do/don't need from it.